app/views/account/dashboard/index.html.slim

```diff
@@ -1,5 +1,143 @@
-.content-body
-  p Hello
-  p
-    = current_user.full_name
+section#js-dashboard
+  javascript:
```

Comment:
try to move out from html, e,g,: `content_for` or place at the bottom ot at the top of dom elements